opentelemetry-sdk/src/opentelemetry/sdk/resources/__init__.py

```diff
@@ -337,12 +335,23 @@ def detect(self) -> "Resource":
                 else sys.version_info,
             )
         )
-
+        _process_pid = os.getpid()
+        _process_executable_name = sys.executable
```

Comment:
[os.getppid](https://docs.python.org/3/library/os.html#os.getppid) for parent process id as well?

Reply:
thank you, i added parent process id